src/components/contact-form/contactForm.js

```diff
@@ -0,0 +1,167 @@
+import { useState } from 'react';
+import styles from './contactForm.module.scss';
+
+const config = {
+  script:
+    'https://script.google.com/macros/s/AKfycbx9XF_VGYdZQoALe4UH3TFyyVMsbENMPMVwp7ZORg/exec',
+  sheet: 'Contact Form Responses',
+  email: 'chanpark93@gmail.com',
+};
+
+const Thanks = ({ name }) => (
+  <div className={styles.thankyou}>
+    <h2>Thanks for contacting us, {name}! We will get back to you soon!</h2>
+  </div>
+);
+
+export default () => {
+  const initialForm = {
+    name: '',
+    organization: '',
+    email: '',
+    phone: '',
+    subject: '',
+    message: '',
+    honeypot: '',
+  };
+
+  const [info, setInfo] = useState(initialForm);
+
+  const { name, organization, email, phone, subject, message, honeypot } = info;
+
+  const initialState = {
+    sent: false,
+    clicked: false,
+  };
+
+  const [state, setState] = useState(initialState);
+
+  const { sent, clicked } = state;
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setInfo({ ...info, [e.target.name]: e.target.value });
+  };
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    // loading
+    setState({ clicked: true });
+
+    //sent
+    const setSent = () => {
+      setState({ sent: true });
```

Comment:
Not sure if this was intentional, but when calling **setState()** for objects, the state will get replaced, not merged.

To mimic **this.setState()** from class components, you would need to merge previous state object:

```js
setState({ ...state, sent: true });
```


> State variables can hold objects and arrays just fine, so you can still group related data together. However, unlike this.setState in a class, updating a state variable always replaces it instead of merging it.

See https://reactjs.org/docs/hooks-state.html#tip-using-multiple-state-variables